jwql/utils/mast_utils.py

```diff
@@ -0,0 +1,291 @@
+"""Various utility functions for interacting with MAST
+
+Authors
+-------
+
+    - Bryan Hilbert
+
+Use
+---
+
+    This module can be imported as such:
+
+    >>> import mast_utils
+    results = mast_utils.mast_query('nircam', 'NRCA1_FULL', 'NRC_DARK', 53005.1, 53005.2)
+
+ """
+import datetime
+import os
+
+from jwql.database.database_interface import Monitor
+from jwql.jwql_monitors import monitor_mast
+from jwql.utils.constants import INSTRUMENT_MONITOR_DATABASE_TABLES, JWST_DATAPRODUCTS
+from jwql.utils.logging_functions import configure_logging, get_log_status
+
+
+def initialize_instrument_monitor(module):
```

Comment:
Weird. I feel like I did that because of circular import issues, although looking at the code now, I don't see any circular imports. I've removed the copies of the functions from mast_utils.py